website/content/en/v0.29/concepts/node-templates.md

```diff
@@ -445,10 +445,6 @@ spec:
 
 For more examples on configuring these fields for different AMI families, see the [examples here](https://github.com/aws/karpenter/blob/main/examples/provisioner/launchtemplates).
 
-{{% alert title="Windows Support Notice" color="warning" %}}
```

Comment:
Likewise, this merge will lead to a feature minor version bump so we shouldn't be updating old versions of the docs with this change.